server/tests/search_providers/belga_360_archive_test.py

```diff
@@ -284,10 +284,10 @@ def test_get_related_article(self):
                 item["firstcreated"], datetime.fromtimestamp(1638953020, utc)
             )
             self.assertEqual(
-                item["versioncreated"], datetime.fromtimestamp(1638952985, utc)
+                item["versioncreated"], datetime(2022, 10, 5, 10, 11, 50, tzinfo=utc)
             )
             self.assertEqual(
-                item["firstpublished"], datetime(2022, 10, 5, 15, 41, 50, tzinfo=utc)
+                item["firstpublished"], datetime(2022, 10, 5, 10, 11, 50, tzinfo=utc)
```

Comment:
time should be rather `13:41:50` if it's converting `20221005154150` from local time to UTC